### DISCUSSION TITLE
rabbitmq-server 3.11.20 bundles osiris 1.5.1?

### DISCUSSION THREAD

**Discussion by VlkrS** [rabbitmq/rabbitmq-server#8910] [Other] [answered]
### Describe the bug

According to the release notes of rabbitmq-server 3.11.19, osiris was updated to 1.6.0, however the source tarball for 3.11.20 (rabbitmq-server-3.11.20.tar.xz) appears to bundle osiris 1.5.1?

### Reproduction steps

Unzip source tarball, check version in ```plugins/osiris-1.5.1/ebin/osiris.app```

### Expected behavior

I would expect version in ```plugins/osiris-1.5.1/ebin/osiris.app```to be 1.6.0 as per the 3.11.19 changelog.

**Comment by michaelklishin** [comment]
This is an issue with release notes.

**Comment by michaelklishin** [comment, marked as answer]
Osiris 1.6 is only used in `main`, and it shipped after 3.11.19, so the issue was certainly with release notes.

**Comment by michaelklishin** [comment]
@VlkrS apparently Osiris 1.6.0 reports its version as 1.5.1. I will keep the release notes as they are because RabbitMQ 3.12 and 3.11 do not use anything 1.6.x-specific anyway.

See https://github.com/rabbitmq/rabbitmq-server/pull/9055 for a follow-up.
